CaliptraChecklistAndEvaluationMethodology.md

```diff
@@ -0,0 +1,303 @@
+  * **Evaluation Methodology:** Manufacturers MUST document the error handling procedures, including what information is reported, how errors are logged, and measures taken to prevent information leakage. Specific error codes and messages should be reviewed for appropriateness.
+
+### *Log Functionality in SoC*
+
+* **Checklist Item:**  
+  * **Requirement:** The SoC SHOULD implement logging functionality that records relevant events related to Caliptra operations, ensuring logs are tamper-evident and provide necessary information for attestation. The log does not need to be kept secure but should be tamper-evident to detect manipulation.  
+  * **Evaluation Methodology:** Manufacturers SHOULD describe the logging mechanism, including how logs are structured, how they are protected against tampering (e.g., using cryptographic hash extensions), and how they integrate with Caliptra's measurement processes.
+
+## Secure Processes
+
+### Development Process
+
+#### *Integrity of Firmware and Hardware*
+
+* **Checklist Item:**  
+  * **Requirement:** The Caliptra RTL MUST match an official release. Netlists, layout, and other development collateral derived from the RTL MUST maintain the same behavior as the released RTL.  
+  * **Evaluation Methodology:** Manufacturers MUST provide evidence of hash verification against official releases. Manufacturers MUST provide evidence that collateral derived from the released RTL has not had its behavior modified.
+
+#### *Caliptra FW Cryptographic Hash*
+
+* **Checklist Item:**  
+  * **Requirement:** Cryptographic hashes of the Caliptra ROM and firmware (FMC and RT) MUST match Caliptra reference hashes of official Caliptra releases.  
+  * **Evaluation Methodology:** Manufacturers MUST provide evidence of hash verification against official releases.
+
+#### *Testing and Verification*
+
+* **Checklist Item:**  
+  * **Requirement:** The product SHOULD undergo thorough testing and verification during development, focusing on both functional correctness and security aspects. This includes testing of specific features and security mechanisms.  
+  * **Evaluation Methodology:** Manufacturers SHOULD present test plans, results, and security analyses covering critical functionalities such as:  
+    * Field programmable entropy API  
+    * Generation and endorsement of attestation messages  
+    * CSR generation and handling  
+    * DICE identity provisioning  
+    * Attestation processes  
+    * Debug state transitions  
+    * SVN updates and enforcement  
+    * Firmware updates (FMC and RT FW)
+
+### Production Process
+
+#### *Debugging Controls*
+
+* **Checklist Item:**  
+  * **Requirement:** The enabling and disabling of debugging features in production MUST be controlled and documented to prevent unauthorized access or leakage of sensitive information. Specific policies regarding when and how debugging can be enabled must be defined.  
```

Comment:
The heading of this section refers to "production process" - initially I read this as "manufacturing process" rather than "system lifecycle in the field". Would it make sense to update the heading?